tutorials/hist/hsumTimer.C

```diff
@@ -12,14 +13,14 @@
 ///
 /// \author Rene Brun
 
-Float_t ratio;
+Float_t ratioratio;
```

Comment:
The variable name ratio is used in a standard library; so when the code is executed outside of a function it raises an exception.